src/status_im/contexts/wallet/events.cljs

```diff
@@ -433,13 +433,15 @@
                                                    [:wallet/bridge-select-token
                                                     (assoc params :network network)]))}])}]]])})))
 
-(rf/reg-event-fx :wallet/start-bridge
- (fn [{:keys [db]}]
-   {:db (assoc-in db [:wallet :ui :send :tx-type] :tx/bridge)
-    :fx [[:dispatch
-          [:wallet/wizard-navigate-forward
-           {:start-flow? true
-            :flow-id     :wallet-bridge-flow}]]]}))
+(rf/reg-event-fx
+ :wallet/start-bridge
+ (fn [{:keys [db]} [{:keys [navigate-forward?]}]]
```

Comment:
I tend to think that it is better to make decision regarding navigation inside the event itself. Because we probably have all the information in the rf-db - current screen, state of the accounts. 
So maybe we can avoid making this decision inside different views and concentrate logic here, wdyt?

Reply:
Absolutely, I've updated the code removing this parameter